public/workspace.json

```diff
@@ -2,18 +2,11 @@
   "workspace": {
```

Comment:
@nerik are the changes in this file intended? maybe we want to have a `galery/` folder and include `custom-gl-layer-workspace.json`?

Reply:
@enriquetuya This is useful just for testing this PR. Maybe this shouldn't even be versioned? I like your idea, but that means maintaining those workspaces...